### PR TITLE
Specify which class's function to call during constructor

### PR DESCRIPTION
**Pull request by AtariDreams**
Especially since the two functions in question are virtual

### Description of the changes:
- Specify to call the superclass's version of the functions during construction, since it is not defined in the subclasses

### How changes were validated:

- Manual testing
- Passed all tests

### CODE REVIEW

**Comment by AtariDreams** [comment]
An updates? @tian-lt

File: src/CalcManager/UnitConverter.cpp [hunk 1]
```diff
@@ -65,8 +65,8 @@ UnitConverter::UnitConverter(_In_ const shared_ptr<IConverterDataLoader>& dataLo
     unquoteConversions[L"{sc}"] = L';';
     unquoteConversions[L"{lb}"] = LEFTESCAPECHAR;
     unquoteConversions[L"{rb}"] = RIGHTESCAPECHAR;
-    ClearValues();
-    ResetCategoriesAndRatios();
+    UnitConverter::ClearValues();
```
Comment (tian-lt) [inline src/CalcManager/UnitConverter.cpp:68]: I agree with Mingxin's comment. Couldn't see the motivation to get rid of polymorphism here.
Could you share more context?

**Review by tian-lt — CHANGES_REQUESTED** [review]

File: src/CalcManager/UnitConverter.cpp (hunk 1 above)
Comment (AtariDreams) [reply src/CalcManager/UnitConverter.cpp:68]: There is only one ClearValues function anyway, despite it being virtual. Polymorphism is not used for this class at all, so either we specify class or these functions should not be virtual

Comment (mingxwa) [reply src/CalcManager/UnitConverter.cpp:68]: Checking the code, `ClearValues()` and `ResetCategoriesAndRatios()` are not declared `virtual` at all. `ClearValues()` has nothing to do with polymorphism. `ResetCategoriesAndRatios` overrides the interface function but does not allow subclasses to override, therefore there's no polymorphism when calling `ResetCategoriesAndRatios()` from `UnitConverter` at runtime.

**Review by mingxwa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**